src/app/elements/element-selection-tab/input-element/input-element-dialog/input-element-dialog.component.ts

```diff
@@ -8,6 +8,7 @@ export interface DialogData {
   selectOptionValues: string[]
   inputValues: string[]
   checkBoxValues: string[]
+  src: string
```

Comment:
What's this used for? I think this can be removed as input doesn't have a field of src. Also don't have checkboxes and select option values, also I think the inputValues can be removed as well.